ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -189,6 +189,7 @@ public class MessageScrubber {
       "firefox-b-o", "firefox-b-1-o", "firefox-b-lm", "firefox-b-1-lm", "firefox-b-lg",
       "firefox-b-huawei-h1611", "firefox-b-is-oem1", "firefox-b-oem1", "firefox-b-oem2",
       "firefox-b-tinno", "firefox-b-pn-wt", "firefox-b-pn-wt-us", "firefox-b-vv", "firefox-b-tf",
+      "firefox-b-1-ar", "firefox-b-ar", "firefox-b-1-dt", "firefox-b-dt",
       "ubuntu", "ubuntu-sn",
```

Comment:
Though I guess CI is complaining it wants it formatted like this:
```suggestion
      "firefox-b-1-ar", "firefox-b-ar", "firefox-b-1-dt", "firefox-b-dt", "ubuntu", "ubuntu-sn",
```